nav2_util/include/nav2_util/angleutils.hpp

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Doesnt ros angles do these operations?

Reply:
Good point. I haven't looked at ros angles. Here I just brought out this section of code from amcl.  We should look into it and see if std library or ros can do these operations.